Casks/t/teleport@16.rb

```diff
@@ -0,0 +1,44 @@
+cask "teleport@16" do
+  version "16.5.9"
+  sha256 "3db8672e2c1b1b4b3b6fc2a4ce31bdaa0164ad944b35cc2d1384df42d1fe6732"
+
+  url "https://cdn.teleport.dev/teleport-#{version}.pkg",
+      verified: "cdn.teleport.dev/"
+  name "Teleport"
+  desc "Modern SSH server for teams managing distributed infrastructure"
+  homepage "https://goteleport.com/"
+
+  livecheck do
+    url "https://goteleport.com/download/"
+    regex(/teleport[._-]v?(16(?:\.\d+)+)\.pkg/i)
+  end
+
+  deprecate! date: "2025-10-01", because: "is EOL"
```

Comment:
Since it'll be ending support this should be 
```suggestion
  disable! date: "2025-10-01", because: : unsupported
```